table-api/src/main/java/io/deephaven/api/updateby/UpdateByOperation.java

```diff
@@ -214,6 +214,141 @@ static UpdateByOperation Ema(OperationControl control, String timestampColumn, D
         return EmaSpec.ofTime(control, timestampColumn, durationDecay).clause(pairs);
     }
 
+
+
+    /**
+     * Create an {@link EmsSpec exponential moving sum} for the supplied column name pairs, using ticks as the decay
+     * unit. Uses the default OperationControl settings.
+     * <p>
+     * The formula used is
+     * </p>
+     *
+     * <pre>
+     *     a = e^(-1 / tickDecay)
+     *     ema_next = a * ema_last + (1 - a) * value
+     * </pre>
```

Comment:
This should be `ems_` and the formula should be different, right? Ditto w/ other javadoc.